.go-version

```diff
@@ -1 +1 @@
-1.70.7
```

Comment:
Are you sure about this? :)

Reply:
wait im so paranoid from screwing this up last time... is it wrong??

Reply:
Sorry! Was just joking given our conversation about the version numbers!  This is :+1:!!!

Reply:
hahaha i thought so but i didnt want to ignore it and screw up again 😅